pkg/strategy/common/profit_fixer.go

```diff
@@ -244,9 +362,19 @@ func fixFromTrades(allTrades []types.Trade, converter *core.ConverterManager, to
 			position.SetFeeAverageCost(trade.FeeCurrency, feePrice)
 		}
 		profit, netProfit, madeProfit := position.AddTrade(trade)
+		stats.AddTrade(trade)
 		if madeProfit {
 			p := position.NewProfit(trade, profit, netProfit)
```

Comment:
I thought NewProfit will load the strategy field and strategy instance id, no ?

Reply:
Yes, you are right.
Fixed.